src/engine/strat_engine/backstore/metadata/bda.rs

```diff
@@ -188,408 +140,16 @@ impl BDA {
-
-    /// Parse a buffer to a StaticHeader.
-    /// Return None if no stratis magic number found.
-    /// Return an error if stored checksum and calculated checksum do not
-    /// match.
-    /// Return an error if the version number is not expected.
-    fn sigblock_from_buf(buf: &[u8]) -> StratisResult<Option<StaticHeader>> {
-        assert_eq!(buf.len(), bytes!(static_header_size::SIGBLOCK_SECTORS));
-
-        if &buf[4..20] != STRAT_MAGIC {
-            return Ok(None);
-        }
-
-        let crc = crc32::checksum_castagnoli(&buf[4..bytes!(static_header_size::SIGBLOCK_SECTORS)]);
-        if crc != LittleEndian::read_u32(&buf[..4]) {
-            return Err(StratisError::Engine(
-                ErrorEnum::Invalid,
-                "header CRC invalid".into(),
-            ));
-        }
-
-        let blkdev_size = BlockdevSize::new(Sectors(LittleEndian::read_u64(&buf[20..28])));
-
-        let version = buf[28];
-        if version != STRAT_SIGBLOCK_VERSION {
-            return Err(StratisError::Engine(
-                ErrorEnum::Invalid,
-                format!("Unknown sigblock version: {}", version),
-            ));
-        }
-
-        let pool_uuid = Uuid::parse_str(from_utf8(&buf[32..64])?)?;
-        let dev_uuid = Uuid::parse_str(from_utf8(&buf[64..96])?)?;
-
-        let mda_size = MDASize(Sectors(LittleEndian::read_u64(&buf[96..104])));
-
-        Ok(Some(StaticHeader {
-            pool_uuid,
-            dev_uuid,
-            blkdev_size,
-            mda_size,
-            reserved_size: ReservedSize::new(Sectors(LittleEndian::read_u64(&buf[104..112]))),
-            flags: 0,
-            initialization_time: LittleEndian::read_u64(&buf[120..128]),
-        }))
-    }
-
-    /// Zero out the entire static header region on the designated file.
-    pub fn wipe<F>(f: &mut F) -> StratisResult<()>
-    where
-        F: Seek + SyncAll,
-    {
-        let zeroed = [0u8; bytes!(static_header_size::STATIC_HEADER_SECTORS)];
-        f.seek(SeekFrom::Start(0))?;
-        f.write_all(&zeroed)?;
-        f.sync_all()?;
-        Ok(())
-    }
-}
-
-impl fmt::Debug for StaticHeader {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        f.debug_struct("StaticHeader")
-            .field("blkdev_size", &self.blkdev_size)
-            .field("pool_uuid", &self.pool_uuid.to_simple_ref())
-            .field("dev_uuid", &self.dev_uuid.to_simple_ref())
-            .field("mda_size", &self.mda_size)
-            .field("reserved_size", &self.reserved_size)
-            .field("flags", &self.flags)
-            .field("initialization_time", &self.initialization_time)
-            .finish()
-    }
-}
-
 #[cfg(test)]
 mod tests {
     use std::io::Cursor;
 
-    use proptest::{collection::vec, num, option, prelude::BoxedStrategy, strategy::Strategy};
-    use uuid::Uuid;
-
-    use devicemapper::{Bytes, Sectors, IEC};
+    use proptest::{collection::vec, num};
 
+    use super::super::sizes::static_header_size;
```

Comment:
Please use crate name and merge the imports, as:

use ```crate::engine::strat_engine::metadata::{sizes::static_header_size, static_header::{tests::random_static_header, tests::static_header_strategy}}```

But always leave the ```use super::*;``` separate in tests, which is the only place we use it.